### PR TITLE
INT-344

### PR DESCRIPTION
**Pull request by maximgubar**

### CODE REVIEW

**Comment by maximgubar** [comment]
@lsmith77 
how do you propose to manage the acl ? I mean who is allowed to remove users ?

**Comment by kichooo** [comment]
10k added oh boy :D

**Comment by lsmith77** [comment]
I don't think ACLs are needed .. in fact right now anyone can delete anyone already ..

**Comment by maximgubar** [comment]
so, I should leave this open to everyone ?

File: web/index.php [hunk 1]
```diff
@@ -93,22 +94,59 @@ function () use ($app, $config, $sqlite) {
         $weeks        = $app['request']->get('view', 1);
         $endDate      = $helper->getEndDate($weeks);
         $days         = $helper->getDays($startDate, $endDate);
-        $calendar     = new GoogleCalendar($config->settings['google'], $startDate, $endDate);
+//        $calendar     = new GoogleCalendar($config->settings['google'], $startDate, $endDate);
```
Comment (syzer) [inline web/index.php]: busted!

File: web/index.php [hunk 2]
```diff
@@ -93,22 +94,59 @@ function () use ($app, $config, $sqlite) {
...
 
         return $app['twig']->render(
             'index.twig',
             array(
                 'teams'   => $sqlite->allTeams(),
                 'persons' => $sqlite->allPersons(),
+//                'isManager' => true
```
Comment (syzer) [inline web/index.php]: do we have managers @liip ? :)

Comment (kichooo) [inline web/index.php]: :trollface:

**Comment by kichooo** [comment]
Holy hell full bootstrap included :dancer:

**Comment by maximgubar** [comment]
I see that there are many people willing to help me on this project, feel free to take any ticket from INT project with the component 'Presence'

**Pull request merged**